app/main/forms.py

```diff
@@ -81,3 +83,7 @@ class ProcessAccountForm(PasswordCheckMixin, forms.Form):
 class ProcessPasswdForm(PasswordCheckMixin, forms.Form):
     passwd = passwd_field()
     passwd_confirm = passwd_confirm_field()
+
+class NewOrgForm(forms.Form):
+    name = forms.CharField(max_length=100, label="Nom de l'association en minuscules")
```

Comment:
This is not a name, this is the ID of the new organization so it should be called "id" or something like that.  Furthermore, I think the label should read "Identififant de l'organisation" with a placeholder like "org-identifier" to hint the admin at the correct format.  We could even use a [field validator](https://docs.djangoproject.com/en/1.8/ref/validators/) to ensure the format of the ID is correct (only `[a-z-]` characters).  If you need help with that, don't hesitate to ask me.